exercises/practice/ocr-numbers/.docs/instructions.md

```diff
@@ -1,79 +1,47 @@
 # Instructions
 
-Given a 3 x 4 grid of pipes, underscores, and spaces, determine which number is represented, or whether it is garbled.
+Optical Character Recognition or OCR is software that converts images of text into machine-readable text.
+Given a grid of characters representing some digits, convert the grid to a string of digits.
+If the grid has multiple rows of cells, the rows should be separated in the output with a `","`.
 
-## Step One
+- The grid is made of one of more lines of cells.
+- Each line of the grid is made of one or more cells.
+- Each cell is three columns wide and four rows high (3x4) and represents one digit.
+- Digits are drawn using pipes (`"|"`), underscores (`"_"`), and spaces (`" "`).
 
-To begin with, convert a simple binary font to a string containing 0 or 1.
+## Edge cases
 
-The binary font uses pipes and underscores, four rows high and three columns wide.
+- If the input is not a valid size, your program should indicate there is an error.
+- If the input is the correct size, but a cell is not recognizable, your program should output a `"?"` for that character.
 
-```text
-     _   #
-    | |  # zero.
-    |_|  #
-         # the fourth row is always blank
-```
+## Examples
 
-Is converted to "0"
-
-```text
-         #
-      |  # one.
-      |  #
-         # (blank fourth row)
-```
-
-Is converted to "1"
-
-If the input is the correct size, but not recognizable, your program should return '?'
-
-If the input is the incorrect size, your program should return an error.
-
-## Step Two
-
-Update your program to recognize multi-character binary strings, replacing garbled numbers with ?
-
-## Step Three
-
-Update your program to recognize all numbers 0 through 9, both individually and as part of a larger string.
-
-```text
- _
- _|
-|_
-
-```
-
-Is converted to "2"
+The following input (without the comments) is converted to `"1234567890"`.
 
 ```text
       _  _     _  _  _  _  _  _  #
-    | _| _||_||_ |_   ||_||_|| | # decimal numbers.
+    | _| _||_||_ |_   ||_||_|| | # Decimal numbers.
     ||_  _|  | _||_|  ||_| _||_| #
-                                 # fourth line is always blank
+                                 # The fourth line is always blank,
 ```
 
-Is converted to "1234567890"
-
-## Step Four
+The following input is converted to `"123,456,789"`.
 
-Update your program to handle multiple numbers, one per line.
-When converting several lines, join the lines with commas.
+<!-- prettier-ignore-start -->
 
 ```text
-    _  _
+    _  _ 
   | _| _|
   ||_  _|
-
-    _  _
-|_||_ |_
+         
```

Comment:
@angelikatyborska I messed up. The CI fails because of the formatting, but somehow I didn't notice it on the PR and I merged it.
The trailing white spaces are the issue here, I suppose we could:
- change the file themselves locally
- change the files upstream
- change our formatting check to ignore docs files (currently we grep for trailing whitespace on every file)
- change my brain so it doesn't mess up again

Since the white spaces are somewhat meaningful here, and it could happen that even more meaningful trailing white spaces are in future docs ([there are more coming here](https://github.com/exercism/problem-specifications/pull/2506/changes)), I'm leaning towards upgrading our format check. WDYT? I mean the last option is tempting, but I'm not sure how to act on it.